config/application.rb

```diff
@@ -27,5 +27,7 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+    config.exceptions_app = self.routes
```

Comment:
Does this do something?

Reply:
I believe it allows us to go to the routes file to render the 404 as opposed to just looking at the file "/public/404"